src/3d/camera.js

```diff
@@ -73,9 +77,13 @@ p5.prototype.camera = function(x, y, z){
  * </div>
  */
 p5.prototype.perspective = function(fovy,aspect,near,far) {
+  var args = new Array(arguments.length);
+  for (var i = 0; i < args.length; ++i) {
+    args[i] = arguments[i];
+  }
```

Comment:
I'd be interested in seeing what this would look like extracted to a module of its own:
```js
var getArgs = require( '../utils/get-arguments' );
getArgs( arguments );
```
or something to that effect. It would reduce the duplicate code substantially